berkeley-function-call-leaderboard/setup.py

```diff
@@ -0,0 +1,40 @@
+from setuptools import setup, find_packages
+import codecs
+
+with codecs.open("README.md", "r", "utf-8") as f:
+    long_description = f.read()
+
+setup(
+    name="bfcl",
+    version="0.1.0",
+    description="Berkeley Function Calling Leaderboard (BFCL)",
+    author="Sky Computing Lab",
+    author_email="sky@cs.berkeley.edu",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    python_requires=">=3.10",
+    license="Apache 2.0",
+    packages=find_packages(include=["bfcl*"]),
+    install_requires=[
```

Comment:
Can we actually have this as a part of "requirements.txt" and then read this file to fill in `install_requires`? Or is this way preferred by everyone?

Reply:
Added a `requirements.txt` file